Doc/whatsnew/3.15.rst

```diff
@@ -121,6 +121,15 @@ Deprecated
 Removed
 =======
 
+http.server
+-----------
+
+* The :class:`!http.server.CGIHTTPRequestHandler` class and
+  the ``--cgi`` flag to the :program:`python -m http.server`
+  command-line interface. Those are deprecated since 3.13.
+  (Contributed by Bénédikt Tran in :gh:`133810`.)
```

Comment:
```suggestion
* Removed the :class:`!http.server.CGIHTTPRequestHandler` class
  and the ``--cgi`` flag to the :program:`python -m http.server`
  command-line interface. Those are deprecated since 3.13.
  (Contributed by Bénédikt Tran in :gh:`133810`.)
```